### ISSUE TITLE
Background color

### ISSUE THREAD

**Issue by textreme** [dai-shi/excalidraw-animate#50]
Is there a way to change the background color of the animation - to transparent or black/dark? I tried changing the background color in the SVG code and then re-exporting it, but the animation became jerky.

**Comment by dai-shi** [comment]
Can you try excalidraw-claymate if it helps?

**Comment by textreme** [comment]
Yes, excalidraw-claymate html export works with a dark background. However, I like the fact that excalidraw-animate exports the animation in SVG. To confirm, excalidraw-animate does not have the ability to export with a transparent/dark background?

**Comment by dai-shi** [comment]
I would need to confirm the code, but I think excalidraw-animate as a tool doesn't support it while excalidraw-animate as a library does.

**Comment by teddyjfpender** [comment]
Also interested in this dark mode feature!

**Comment by dai-shi** [comment]
Have you tried excalidraw-claymate? It has more abilities like changing the animation speed and order.

**Comment by dai-shi** [comment]
Is this solved in the latest version of excalidraw-animate?
